src/types/index.js

```diff
@@ -2,4 +2,4 @@ module.exports = {
   value: require('./value'),
```

Comment:
This one got left behind. I don't know if that was intentional.

Reply:
Thanks not intentional